app/controllers/template_options_controller.rb

```diff
@@ -47,12 +47,14 @@ def index
       # Retrieve the Org's templates
       @templates << Template.published.organisationally_visible.where(org_id: org.id,
                                                                       customization_of: nil).sort_by(&:title).to_a
-      @templates = @templates.flatten.uniq
     else
       # if'No Primary Research Institution' checkbox is checked,
       # only show publicly available template without customization
-      @templates = Template.published.publicly_visible.where(org_id: funder.id, customization_of: nil)
+      @templates << Template.default if Template.default.present?
+      @templates << Template.published.publicly_visible.where(org_id: funder.id, customization_of: nil,
+                                                              is_default: false).sort_by(&:title).to_a
     end
+    @templates = @templates.flatten
```

Comment:
I would've called `@templates = @templates.flatten.uniq` at the end of the else block. Because it already exists in the if block as well, I just movement it outside of the block to avoid redundancy (and the rubocop offense).

I also changed `@templates = @templates.flatten.uniq` to `@templates = @templates.flatten`. `@templates = @templates.uniq` gets called at the end of the controller action, so calling `.uniq` again on line 57 would have been redundant.